myblog/views.py

```diff
@@ -0,0 +1,42 @@
+from django.http import HttpResponse, JsonResponse
+
+from myblog.models import Post, PostView
+
+
+def getPosts(request):
+    posts_to_return = list();  # Make an empty list
+
+    m = 0
+    while True:
```

Comment:
If you couldn't use a serializer, instead of using a while loop, it would be better to loop through the collection rather than keeping an index and breaking when the index goes beyond the collection.